src/components/dialog-full-screen/dialog-full-screen.js

```diff
@@ -102,14 +102,14 @@ class DialogFullScreen extends Modal {
    * Overrides the original function to disable the document's scroll.
    */
   get onOpening() {
-    this.document.documentElement.classList.add(DIALOG_OPEN_HTML_CLASS);
+    return this.document.documentElement.classList.add(DIALOG_OPEN_HTML_CLASS);
   }
 
   /**
    * Overrides the original function to enable the document's scroll.
    */
   get onClosing() {
-    this.document.documentElement.classList.remove(DIALOG_OPEN_HTML_CLASS);
+    return this.document.documentElement.classList.remove(DIALOG_OPEN_HTML_CLASS);
```

Comment:
Is this a problem with it being a get method rather than its not returning

Reply:
correct - would need a larger refactor to resolve properly